frontend/src/styles/styles.css

```diff
@@ -0,0 +1,13 @@
+:root {
+    --main-bg-color: rgb(180, 41, 125);
+}
+
+body {
+    font-family: sans-serif;
+    background-color: var(--main-bg-color);
+    margin: 0;
```

Comment:
Often you want to add a proper "css reset"

Reply:
iknoow, but is it necessary every time?

Reply:
it's not _necessary_, it makes things much easier in the long run.